README.md

```diff
@@ -1,64 +1,213 @@
+
+
+Use the [`xrpl.transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html) module to sign and submit transactions. The module offers three ways to do this:
+
+* [`safe_sign_and_submit_transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.safe_sign_and_submit_transaction) — Signs a transaction locally, then submits it to the XRP Ledger. This method does not implement [reliable transaction submission](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission) best practices, so only use it for development or testing purposes.
+
+* [`safe_sign_transaction`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.safe_sign_transaction) — Signs a transaction locally. This method **does  not** submit the transaction to the XRP Ledger.
+
+* [`send_reliable_submission`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.transaction.html#xrpl.transaction.send_reliable_submission) — An implementation of the [reliable transaction submission guidelines](https://xrpl.org/reliable-transaction-submission.html#reliable-transaction-submission), this method signs a transaction locally, submits the transaction to the XRP Ledger, and then verifies that it has been included in a validated ledger (or has failed to do so). Use this method to submit transactions for production purposes.
+
+
+```py
+from xrpl.models.transactions import Payment
+from xrpl.models.transactions.transaction import Transaction
+from xrpl.transaction import send_reliable_submission
+from xrpl.ledger import get_fee
+
+FEE = get_fee(JSON_RPC_CLIENT)
 
-```bash
-poetry run python -m unittest discover tests.integration.reliable_submission
+# prepare the transaction
+my_tx_payment = Payment(
+    account=TESTNET_CLASSIC_ACCOUNT,
+    amount=_AMOUNT,
+    destination=TESTNET_DESTINATION_ACCOUNT,
+    signing_pub_key=TESTNET_WALLET.pub_key,
+    last_ledger_sequence=TESTNET_WALLET.next_sequence_num + 10,
+    sequence=TESTNET_WALLET.next_sequence_num,
+    fee=FEE,
+)
+# sign the transaction
+my_tx_payment_signed = safe_sign_transaction(my_tx_payment,TESTNET_WALLET)
+
+# submit the transaction
+tx_response = send_reliable_submission(my_tx_payment, TESTNET_WALLET, JSON_RPC_CLIENT_TESTNET)
 ```
 
-### Generating Documentation
+### Encode addresses
 
-From the `docs` folder,
+Use the [`xrpl.core.addresscodec`](https://xrpl-py.readthedocs.io/en/latest/source/xrpl.core.addresscodec.html) to encode and decode addresses into the "classic" and X-address formats.
 
-```bash
-poetry run sphinx-apidoc -o source/ ../xrpl
-poetry run make html
+```py
+# convert classic address to x-address
+>>> from xrpl.core import addresscodec
+>>> TESNET_XADDRESS = addresscodec.classic_address_to_xaddress(TESTNET_CLASSIC_ACCOUNT, False, True)
```

Comment:
The second argument is supposed to be the tag (an integer from 0 to 2³² inclusive), not a boolean, if you provide it. Changing to keyword arguments makes this section much easier to understand, and using hardcoded values here will make this code snippet stand alone (again, for the benefit of people skimming):

```suggestion
>>> testnet_xaddress = addresscodec.classic_address_to_xaddress("ra9cjuPRq7tmR3Uin3y6fUPKLZ317JLvbN", tag=0, is_test_network=True)
```